README.md

```diff
@@ -0,0 +1,161 @@
+- [x] **Manage transaction summaries**
+- [x] **Add/remove transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="built-with"></a>
+
+[Demo Link](https://www.loom.com/share/c3dcc2d054974090b7f43d96b8776643?sid=c6c0493f-f4e4-40d5-baaa-1935c64ed509)
+
+## Deployment
+
+[Link](https://budgetapp-mc2o.onrender.com/)
+
+## Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+  * A basic understanding of Ruby, Rails and Git
+### Setup
+
+```
+git clone git@github.com:Katended/budget-app.git
+```
+### Install
+
+1. Install Ruby
+2. Install Rails
+3. Clone this repo
+
+### Usage
+To run the project, execute the following command:
+
+```
+  rails s
+```
+### Run Tests
+To run tests, run the following command:
+
+```
+bundle exec rspec spec
+
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **David Katende**
+
+- GitHub: [@Katended](https://github.com/Katended)
+- LinkedIn: [@katendedavid](https://www.linkedin.com/in/katendedavid/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Modern UI**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Katended/budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please support this project and be a part of something meaningful and impactful. Your contribution can make a significant difference and help us achieve our goals faster.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Grateful to all for inspiring codebases, your dedication and creativity made a lasting impact on our success. 
+
+Also special thanks goes to  [Gregoire Vella](https://www.behance.net/gregoirevella) for his insperational design of this project.
+
+Thank you!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## FAQ <a name="faq"></a>
+
+- **Can I fork and reuse this repository?**
+
+Absolutely! Please feel free to fork and reuse this repository for your own projects.
+
+- **Is it okay to improve this repository? Will my changes be accepted?**
+
+Definitely! We welcome any improvements or new ideas you may have. If your changes are good and align with the project's goals, we would be happy to accept them.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] just as previous reviewer mentioned, your MIT link is still broken, kindly fix that

Reply:
@VICTOR-CHUKWU ,

I need some help on this because I can not identify the issue. It seems to work well on my side after the previous review fix. Can you please hop on zoom or something?